common/src/main/java/org/ergoplatform/appkit/ErgoType.java

```diff
@@ -13,27 +13,27 @@
  * wrapper around {@link RType} type descriptor.
  */
 public class ErgoType<T> {
-    private static ErgoType<Byte> _byte = new ErgoType<>(RType.ByteType());
-    private static ErgoType<Short> _short = new ErgoType<>(RType.ShortType());
-    private static ErgoType<Integer> _integer = new ErgoType<>(RType.IntType());
-    private static ErgoType<Long> _long = new ErgoType<>(RType.LongType());
-    private static ErgoType<BigInt> _bigInt = new ErgoType<>(JavaHelpers.BigIntRType());
-    private static ErgoType<GroupElement> _groupElement = new ErgoType<>(JavaHelpers.GroupElementRType());
-    private static ErgoType<SigmaProp> _sigmaProp = new ErgoType<>(JavaHelpers.SigmaPropRType());
-    private static ErgoType<AvlTree> _avlTree = new ErgoType<>(JavaHelpers.AvlTreeRType());
-    private static ErgoType<Header> _header = new ErgoType<>(JavaHelpers.HeaderRType());
-    private static ErgoType<PreHeader> _preHeader = new ErgoType<>(JavaHelpers.PreHeaderRType());
+    private static ErgoType<scala.Byte> _byte = new ErgoType(RType.ByteType());
+    private static ErgoType<scala.Short> _short = new ErgoType(RType.ShortType());
+    private static ErgoType<scala.Int> _integer = new ErgoType(RType.IntType());
```

Comment:
Could this break existing code using Appkit?

Reply:
@scalahub Maybe, but the fixes should be straightforward, and this is actually fixes a potential bugs.